requirements.txt

```diff
@@ -0,0 +1,4 @@
+mailru-im-bot==0.0.21
+requests==2.26.0
+retry==0.9.2
+flake8==6.0.0
```

Comment:
It is a dependency for testing. Looks like it shouldn't be here.

Reply:
Fixed

Reply:
You should remove that in this commit, not in the second one.